src/python/pants/core_tasks/deferred_sources_mapper.py

```diff
@@ -14,13 +14,14 @@
 from pants.engine.fs import PathGlobs, PathGlobsAndRoot
 from pants.source.wrapped_globs import EagerFilesetWithSpec
 from pants.task.task import Task
+from pants.task.unpack_remote_sources_base import UnpackedArchives
 
 
 logger = logging.getLogger(__name__)
 
 
 class DeferredSourcesMapper(Task):
-  """Map DeferredSourcesFields to files that produce the product 'unpacked_archives'.
+  """Map DeferredSourcesFields to files that produce the product `UnpackedArchives`.
```

Comment:
FTR: `DeferredSourcesField` was deprecated and removed in favor of the `remote_sources` target type. Sorry that this is so hairy.